aptos-move/aptos-aggregator/src/aggregator_change_set.rs

```diff
@@ -4,49 +4,49 @@
 use crate::{
     bounded_math::code_invariant_error,
     delta_change_set::DeltaOp,
-    types::{AggregatorID, AggregatorValue, SnapshotToStringFormula},
+    types::{AggregatorValue, SnapshotToStringFormula},
 };
 use move_binary_format::errors::PartialVMResult;
 
 #[derive(Clone, Debug, Eq, PartialEq)]
-pub enum AggregatorApplyChange {
+pub enum AggregatorApplyChange<I: Clone> {
```

Comment:
Why does the `AggregatorID` have to be a generic type? AggregatorID is always going to be a u64 right?